sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/SyncSample.java

```diff
@@ -34,11 +35,11 @@ public static void main(String[] args) throws JsonProcessingException {
             .buildClient();
 
         // Create relationship on a digital twin
-        String createdRelationship = client.createRelationshipWithResponse(sourceDigitalTwinId, relationshipId, relationship).getValue();
+        String createdRelationship = client.createRelationshipWithResponse(sourceDigitalTwinId, relationshipId, relationship, Context.NONE).getValue();
```

Comment:
So is this treated as empty?

Reply:
yes
`public static final Context NONE = new Context((Context)null, (Object)null, (Object)null);`